shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -80,15 +82,18 @@ bool TizenViewElementary::CreateView() {
   }
   evas_object_size_hint_weight_set(container_, EVAS_HINT_EXPAND,
                                    EVAS_HINT_EXPAND);
+  EvasObjectResizeWithMinMaxHint(container_, initial_width_, initial_height_);
 
   image_ = evas_object_image_filled_add(evas_object_evas_get(container_));
   if (!image_) {
     FT_LOG(Error) << "Failed to create an Evas object image.";
     return false;
   }
   evas_object_size_hint_align_set(image_, EVAS_HINT_FILL, EVAS_HINT_FILL);
+  EvasObjectResizeWithMinMaxHint(image_, initial_width_, initial_height_);
```

Comment:
This partially reverts https://github.com/flutter-tizen/engine/commit/4901d942c33816c4cdf78c9d9f244b9e513ec2a4. Is this intentional? I thought `evas_object_size_hint_align_set` would take care of the resizing automatically.

Reply:
This change has been made because `SetGeometry` no longer handle image_.

Based on my understanding, I expected that `evas_object_size_hint_align_set` should affect layout and child would follow the size of the parent if `EVAS_HINT_EXPAND` was used for child. 
However, the actual behaviors were slightly different. the initial size was not set as I expected.  after that, when changing the size, the size of the child was adjusted to the size of the parent as I expected.
so, I kept the logic of using hints.